client_py/test_client.py

```diff
@@ -0,0 +1,20 @@
+from unittest import TestCase
+from client_py.client import QueueClient
+
+
+class TestQueueClient(TestCase):
```

Comment:
Why we have `client_test.py` and `test_client.py`. Please remove the redundent one and only keep one.